### PR TITLE
feature/WEBST-250 Sent and share arrow icons

### PR DESCRIPTION
**Pull request by sara-tustra**
This PR adds sent and share arrow icons required in blog views.

### CODE REVIEW

File: src/react/_icons/ShareArrow.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Icon from '../components/Icon';
+import { IconComponent } from '../types';
+
+const ShareArrow: IconComponent = ({ fill = 'currentColor', stroke = 'currentColor', ...props }) => (
+    <Icon viewBox="0 0 24 24" {...props}>
+        <g fill="currentColor">
```
Comment (panz0rr) [inline src/react/_icons/ShareArrow.tsx:7]: This fill should be in the Icon component and g must be deleted (as that is the role of the Icon)

**Review by panz0rr — CHANGES_REQUESTED** [review]

File: src/react/_icons/ShareArrow.tsx (hunk 1 above)
Comment (sara-tustra) [reply src/react/_icons/ShareArrow.tsx]: Done!

**Review by panz0rr — APPROVED** [review]

**Review by raulWTW — APPROVED** [review]
Correct name convention for PR

**Review by Tisks — APPROVED** [review]
In this case the PR name would be

feature/WEBST-250 Sent and share arrow icons

**Pull request merged**